test/gtest/ucs/test_async.cc

```diff
@@ -603,6 +641,34 @@ UCS_TEST_P(test_async, warn_block) {
     }
 }
 
+#if UCS_ENABLE_ASSERT
```

Comment:
why needed?

Reply:
gtest can be compiled in release mode where this test will fail

Reply:
why would it fail? check_is_blocked is already NOP in release mode

Reply:
need to use `check_is_blocked` here - done